### PR TITLE
use standard GitHub Actions debugging

### PR DESCRIPTION
**Pull request by rindeal**
#### Summary

I don't see the need for wheel reinvention.

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/troubleshooting-workflows/enabling-debug-logging#enabling-step-debug-logging

The workflow after this change is basically: if a job fails, you re-run it with debugging enabled and see all the debug logs.

#### Release Note

NONE

#### Documentation

NONE

### CODE REVIEW

**Comment by woodruffw** [comment]
This is an internal option that we explicitly don't document as part of a public API. The reason it doesn't use the built-in actions debugging state is because that state requires the user to change a workflow-level variable or secret, which both requires a larger context switch (moving from the editor to the repo's settings on GitHub) and also assumes that workflow re-runs are idempotent (which in the context of release signing, they typically aren't).

TL;DR: There's a reason this is done the way it is. I'd be happy to review a PR that *adapts* the current setting to also take `ACTIONS_STEP_DEBUG` into account as a default, but removing it outright is not something that I think we're likely to do in the short term.
